images/vm-image-builder/Dockerfile

```diff
@@ -9,3 +9,5 @@ RUN dnf install -y \
     virt-install && \
     dnf clean all && \
     rm -rf /var/cache /var/log/dnf* /var/log/yum.*
+
+RUN ln -s /usr/bin/mkisofs /usr/bin/genisoimage
```

Comment:
This looks wrong. If you want the `genisoimage` command to be available, just install the `genisoimage` package along with the other ones.

But even that shouldn't be needed, because `virt-install` drags in the `xorriso` package and [has code in it](https://github.com/virt-manager/virt-manager/blob/102fe52165535f9da9c9f370c6d1c399458ed4dc/virtinst/install/installerinject.py#L53-L61) specifically trying `xorrisofs`, `genisoimage` and `mkisofs` in order.

Can you please provide the exact error message that you're seeing?

Reply:
```
cloud-localds cloudinit.iso /workspace/cluster-provision/images/vm-image-builder/example/cloud-config
missing 'genisoimage'.  Required for --filesystem=iso9660.
```